impeller/aiks/experimental_canvas.cc

```diff
@@ -375,8 +381,9 @@ void ExperimentalCanvas::SaveLayer(
           return filter;
         };
 
-    auto input_texture = FlipBackdrop(render_passes_, GetGlobalPassPosition(),
-                                      clip_coverage_stack_, renderer_);
+    auto input_texture =
```

Comment:
When doing the clip replay for a bdf, what value should be used as the clip depth comparison? @bdero

Reply:
Apologies for the late reply. When doing a clip restore:
- The MSAA backdrop should be the _very first thing_ drawn, but I set the clip depth to `std::numeric_limits<uint32_t>::max()` anyway.
- Afterwards, each clip being replayed should have _the same depth_ they had the first time they were being drawn.

Reply:
Updated this to use max int32 for bdf restores, other flip backdrops (emulated advanced blend) use the entity depth